From 2e6187f104c762bf9e87f78dbd7f3a2dbe088b99 Mon Sep 17 00:00:00 2001
From: Peter Tworek <tworaz666@gmail.com>
Date: Tue, 14 Apr 2015 21:17:33 +0200
Subject: [PATCH 8/9] Add const keyword to WebCursor::GetNativeCursor()

The function should not actually modify WebCursor instance. Since
content layer often gives us const WebCursor& the function is not usable
anyway.
---
 content/common/cursors/webcursor.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/content/common/cursors/webcursor.h b/content/common/cursors/webcursor.h
index cd78c58..b12e21c 100644
--- a/content/common/cursors/webcursor.h
+++ b/content/common/cursors/webcursor.h
@@ -94,7 +94,7 @@ class CONTENT_EXPORT WebCursor {
   bool IsEqual(const WebCursor& other) const;
 
   // Returns a native cursor representing the current WebCursor instance.
-  gfx::NativeCursor GetNativeCursor();
+  gfx::NativeCursor GetNativeCursor() const;
 
 #if defined(OS_WIN)
   // Initialize this from the given Windows cursor. The caller must ensure that
-- 
2.0.5

